Makefile

```diff
@@ -0,0 +1,15 @@
+CLIPPY_OPTIONS := -- -D warnings -D clippy::redundant_clone -D clippy::enum_glob_use \
```

Comment:
Using a config file is more preferable https://doc.rust-lang.org/stable/clippy/configuration.html.